src/containers/shared/css/variables.scss

```diff
@@ -57,6 +58,7 @@ $testnet: #e54d00;
 $devnet: #008ae5;
 $nft-sandbox: #791aff;
 $sidechain: #faff1a;
+$amm: $magenta-600;
```

Comment:
Why is this defined in 2 steps?

Reply:
The color came from design.  I wanted to separate the color variable and is usage.  Design kept calling it magenta-600.  Trying to flesh out the colors some more to be inline with xrpl.orgs